src/Common/tests/TestUtilities/CommonTestHelper.cs

```diff
@@ -188,9 +188,9 @@ public static TheoryData<string> GetStringTheoryData()
             return data;
         }
 
-        public static TheoryData<string> GetStringWithNullTheoryData()
+        public static TheoryData<string?> GetStringWithNullTheoryData()
         {
-            var data = new TheoryData<string>
+            var data = new TheoryData<string?>
```

Comment:
as we are adapting following notion, you may want to make those changes as well for the lines/files your touch.
```suggestion
            TheoryData<string?>data = new
```